### PR TITLE
Bump to v2.0.1

### PR DESCRIPTION
**Pull request by lcbm**
(Thanks for sending a pull request! Please fill out the blanks below.)

What does this implement/fix? Explain your changes.
---------------------------------------------------

This pull request adds all notable changes made since the last release in the CHANGELOG file, and bumps the current project version to v2.0.1.

Does this close any currently open issues?
------------------------------------------
(Use the special github keywords to reference issues: https://help.github.com/en/articles/closing-issues-using-keywords)

Nope! :smiley: 

Any relevant logs, error output, etc?
-------------------------------------
(If it’s long, please paste to https://pastebin.com/ and insert the link here.)

Any other comments?
-------------------

Nope! :smiley: 

Where has this been tested?
---------------------------
(Describe your environment setup here.)

**Operating System/Platform:** Linux 5.4.6-arch1-1 x86_64 GNU/Linux
**NPM Version:** v13.5.0 (via [nvm](https://github.com/nvm-sh/nvm) v0.35.2)
**NodeJS Version:** 6.13.4  (via [nvm](https://github.com/nvm-sh/nvm)  v0.35.2)

(Add more information here if needed.)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# [2.0.1](https://github.com/CESARBR/knot-fog-connector-knot-cloud/compare/v2.0.0...v2.0.1)
+
+### Bug Fixes
+
+- update dependency lodash to v4.17.13 ([#10](https://github.com/CESARBR/knot-fog-connector-knot-cloud/pull/10)) ([cdafff9](https://github.com/CESARBR/knot-fog-connector-knot-cloud/pull/10/commits/cdafff964152978a2f0b766322b8c79b71afd5f2))
```
Comment (joaoaneto) [inline CHANGELOG.md]: Let start with a capital letter like the others. Also, I think you can just put "Update dependencies to fix vulnerability issues" and the packages would go as sub-items, without specifying the commit they were updated. The reason is that if we start to do this manually, it'd be difficult to create the CHANGELOG for the next releases. We can solve this problem by using a tool to automate this process as I've investigated before. Do you agree?

Comment (lcbm) [reply CHANGELOG.md]: Yeah, totally! :smile: 
Here are some tools we could consider, actually :point_down: 

[standard-version](github.com/conventional-changelog/standard-version): standard-version, “automates versioning and CHANGELOG generation, with semver.org and conventionalcommits.org.

[semantic-release](semantic-release.gitbook.io/semantic-release): automates the whole package release workflow including: determining the next version number, generating the release notes and publishing the package.

Comment (joaoaneto) [reply CHANGELOG.md]: Yes, I know our CHANGELOG version could be improved according to these guidelines but I think we should keep as it is as a matter of time. However, about the sub-items point, you can go with the approach you suggested but starting with a capital letter.

Comment (lcbm) [reply CHANGELOG.md]: Oops! Fixed capitalization :smile:

**Pull request merged**